### ISSUE TITLE
Problem when use nested router with route path `/`

### ISSUE THREAD

**Issue by gaconkzk** [pateketrueke/yrv#55]
Seem we can't use nested route with default '/' path, do we have any plan or work around for this case?

Example: main router
```
<Router>
  <Route path="/" component="{Home}" />
  <Route path="/login" exact component="{Login}" />
</Router>
```

Then in Home component we also have a router

```
<Router>
  <Route exact component="{Dashboard}" />
  <Route path="/:id" exact component="{DetailItem}" />
</Router>
```
I tried many combination, including add/remove exact prop, but seem the router render is not working as I want, when use exact in path "/", the child router not rendering at all.
When clear exact, the /login will also render the Home component.

For now,  I have to use `/main` or `/home` for the Home component instead of `/` that I actually need :(

**Comment by pateketrueke** [comment]
So much thanks for your input, I'll need some time to investigate this, probably it align with other issues we have.

**Comment by pateketrueke** [comment]
This seems to be fixed on `v0.0.57` 💣

**Comment by pateketrueke** [comment]
This seems to be fixed on `v0.0.57` 💣
